docs/resources/cloud_credential.md

```diff
@@ -120,3 +120,29 @@ The following attributes are exported:
 - `create` - (Default `10 minutes`) Used for creating cloud credentials.
 - `update` - (Default `10 minutes`) Used for cloud credential modifications.
 - `delete` - (Default `10 minutes`) Used for deleting cloud credentials.
+
+## Import
+
+Node Template can be imported using the Rancher Node Template ID
+
+```bash
+terraform import rancher2_cloud_credential.foo &lt;cloud_credential_id&gt;.&lt;driver&gt;
```

Comment:
For docs notation consistency, could you please set the variable names is capital letters??